test/CrossImport/horrible.swift

```diff
@@ -0,0 +1,44 @@
+// RUN: %empty-directory(%t)
+// RUN: cp -r %S/Inputs/lib-templates/ %t/
+
+// RUN: %target-typecheck-verify-swift -enable-cross-import-overlays -I %t/lib/swift
```

Comment:
Could you modify this test case (or add another one, whichever is easier) to make sure cross-import overlays are working with the trace files and `.d` files?

Reply:
CrossImport/loading.swift tests `-emit-dependencies-path`, and CrossImport/module-trace.swift tests `-emit-loaded-module-trace-path`.

Reply:
Oops, those are in a later commit. Sorry!